odl/contrib/datasets/images/cambridge.py

```diff
@@ -0,0 +1,151 @@
+URL_CAM = 'http://store.maths.cam.ac.uk/DAMTP/me404/data_sets/'
+
+
+def brain_phantom(size=None, to_odl=True):
+    """Brain phantom for FDG PET simulations.
+
+    Output:
+        image type: gray scales
+        size: [1024, 1024] (if not specified by `size`)
+        scale: [0, 1]
+        type: float64
+    """
+    # TODO: Store data in some ODL controlled url
+    name = 'PET_phantom.mat'
+    url = URL_CAM + name
+    dct = get_data(name, subset=DATA_SUBSET, url=url)
+    im = np.rot90(dct['im'], k=3)
+
+    return convert(im, size, to_odl)
+
+
+def resolution_phantom(size=None, to_odl=True):
+    """Resolution phantom for tomographic simulations.
+
+    Output:
+        image type: gray scales
+        size: [1024, 1024] (if not specified by `size`)
+        scale: [0, 1]
+        type: float64
+    """
+    # TODO: Store data in some ODL controlled url
+    # TODO: This can be also done with ODL's ellipse_phantom
+    name = 'phantom_resolution.mat'
+    url = URL_CAM + name
+    dct = get_data(name, subset=DATA_SUBSET, url=url)
+    im = np.rot90(dct['im'], k=3)
+
+    return convert(im, size, to_odl)
+
+
+def building(size=None, gray=False, to_odl=True):
+    """Photo of the Centre for Mathematical Sciences in Cambridge.
+
+    Output:
+        image type: color (or gray scales if `gray=True`)
+        size: [442, 331] (if not specified by `size`)
+        scale: [0, 1]
+        type: float64
+    """
+    # TODO: Store data in some ODL controlled url
+    name = 'cms.mat'
+    url = URL_CAM + name
+    dct = get_data(name, subset=DATA_SUBSET, url=url)
+    im = np.rot90(dct['im'], k=3)
+
+    return convert(im, size, to_odl, gray=gray)
+
+
+def rings(size=None, gray=False, to_odl=True):
+    """Photo of married couple holding hands.
+
+    Output:
+        image type: color (or gray scales if `gray=True`)
+        size: [3264, 2448] (if not specified by `size`)
+        scale: [0, 1]
+        type: float64
+    """
+    # TODO: Store data in some ODL controlled url
+    name = 'rings.mat'
+    url = URL_CAM + name
+    dct = get_data(name, subset=DATA_SUBSET, url=url)
+    im = np.rot90(dct['im'], k=2)
+
+    return convert(im, size, to_odl, gray=gray)
+
+
+def blurring_kernel(size=None, to_odl=True):
+    """Blurring kernel for convolution simulations.
+
+    The kernel is scaled to sum to one.
+
+    Output:
+        image type: gray scales
+        size: [100, 100] (if not specified by `size`)
+        scale: [0, 1]
+        type: float64
+
+    """
+    # TODO: Store data in some ODL controlled url
+    name = 'motionblur.mat'
+    url = URL_CAM + name
+    dct = get_data(name, subset=DATA_SUBSET, url=url)
+
+    return convert(255 - dct['im'], size, to_odl, normalize='sum')
+
+
+def convert(image, size, to_odl, gray=None, dtype='float64', normalize='max'):
+
+    image.astype(dtype)
+
+    if gray is not None and gray:
+        image = np.mean(image, 2)
+
+    if size is not None:
+        image = imresize(image, size)
```

Comment:
Typically `size` is a number (e.g. the number of pixels in the image) while `shape` is the number of pixels per axis, I guess we should change `size` -> `shape` here